stories/atoms/components/icon/icon.scss

```diff
@@ -14,6 +14,12 @@
   }
 }
 
+.invert {
+  @include with-theme() {
+    background: themed('field-high-contrast');
```

Comment:
OBS #492 will change the name of this background

Reply:
Yes, the fix will be made with the rebasing